test/test_models.py

```diff
@@ -783,6 +793,61 @@ def check_out(out):
     _check_input_backprop(model, x)
 
 
+def check_model_output(out, model_name):
+    assert len(out) == 1
+
+    def compact(tensor):
+        tensor = tensor.cpu()
+        size = tensor.size()
+        elements_per_sample = functools.reduce(operator.mul, size[1:], 1)
+        if elements_per_sample > 30:
+            return compute_mean_std(tensor)
+        else:
+            return subsample_tensor(tensor)
+
+    def subsample_tensor(tensor):
+        num_elems = tensor.size(0)
+        num_samples = 20
+        if num_elems <= num_samples:
+            return tensor
+
+        ith_index = num_elems // num_samples
+        return tensor[ith_index - 1 :: ith_index]
+
+    def compute_mean_std(tensor):
+        # can't compute mean of integral tensor
+        tensor = tensor.to(torch.double)
```

Comment:
is there a reason for using `torch.double` instead of `torch.float32`?
also, just in case there exists `torch.std_mean` function

Reply:
Thanks for reviewing @vadimkantorov . This function used to be an inner function of `test_detection_model()` and I just copied it outside the function so that other functions can call it. So I don't know if there has been a reason to cast it to double instead of float32, but based on the above comment it seems like float32 would be fine. Should I change it to float32 and also switch to `std_mean()`?

Reply:
I'm an not one of maintainers, let's wait for a more official review :)

Reply:
@vadimkantorov all right, thanks for pointing that out though.